src/panel/direction/DirectionForm.jsx

```diff
@@ -30,10 +30,15 @@ export default class DirectionForm extends React.Component {
     if (isMobileDevice() || this.props.isInitializing) {
       return;
     }
-    const { origin, destination, originInputText, destinationInputText } = this.props;
-    if (!originInputText && (prevProps.destination !== destination || prevProps.isInitializing)) {
+
+    const { origin, destination } = this.props;
+
+    if (!origin && destination &&
+        (prevProps.destination !== destination || prevProps.isInitializing)) {
+      // a destination is set, origin is empty, so let's focus on origin
       this.originRef.current.focus();
```

Comment:
Thanks for the comments !  
But we still want to focus the origin field when both fields are empty, right ?

Reply:
This is still the case using `prevProps.isInitializing`, if page is "reloaded".